tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -29,6 +29,7 @@
 from azure.mgmt.web.models import NameValuePair
 from c7n_azure import constants
 from c7n_azure.constants import RESOURCE_VAULT
+from future.moves import itertools
```

Comment:
this wasn't needed incidentally, the import path for this module and the accessed function is the same across 2.7/3

Reply:
found while investigating why this pr errr'd on trunk docker build
https://cloud.docker.com/u/cloudcustodian/repository/docker/cloudcustodian/c7n/builds/27e684a8-78ea-4e8d-9666-10240fed4504